compose.yaml

```diff
@@ -48,6 +48,7 @@ services:
       - --system-parameter-default=max_clusters=100
       - --system-parameter-default=max_sources=100
       - --system-parameter-default=max_aws_privatelink_connections=10
+      - --system-parameter-default=transaction_isolation=serializable
```

Comment:
Setting the transaction isolation level to serializable for the second region, that way we will make sure that even if the default value is not set to strict serializable, everything will still work as the provider will explicitly set the level to strict serializable.

Reply:
Smart!